tests/src/FunctionalJavascript/CorporateContactFormTest.php

```diff
@@ -172,6 +179,22 @@ public function testCorporateContactForm(): void {
     // Assert fields are now visible.
     $this->assertFieldsVisible(TRUE);
 
+    // Assert alternative contact language is active only when preferred
+    // contact language field is active.
```

Comment:
We should test that it is not available for non-corporate forms.

Reply:
Also I don't see the case when we have Preferred checkbox saved the Alternative checkbox is not disabled on page load.